src/utils.jl

```diff
@@ -216,6 +216,7 @@ function degree(p::Term, sym=nothing)
 end
  
 function degree(p, sym=nothing)
+    @assert !(typeof(p) <: Array) "Input may not be an Array."
```

Comment:
That was just an example. I was trying to say that you need to explicit check `p isa Symbolic` before calling the other methods.

Reply:
I see what you mean. Shouldn't it check instead for `p` being in a union of `Num` and `Number`?

Reply:
`p` is already unwrapped by `value`, so it won't be a `Num`.